### ISSUE TITLE
Place=suburb should maybe be rendered at higher zoom levels

### ISSUE THREAD

**Issue by fqdb** [gravitystorm/openstreetmap-carto#2546]
In most places around the world, a suburb is an area with a significant population, which may be much higher than that of a town. Currently, suburb labels are rendered from level 12 and up, whereas town names are rendered from level 8 already. In zoom level, suburbs appear at the same time as villages, which obviously have less importance in most cases.

In contrast, the transport layer renders suburb labels from level 10 already, which looks much more balanced.

Or am I interpreting the meaning of place=suburb wrong? For example,[in this case](http://overpass-turbo.eu/map.html?Q=%2F*%0AThis%20has%20been%20generated%20by%20the%20overpass-turbo%20wizard.%0AThe%20original%20search%20was%3A%0A%E2%80%9Cplace%3Dsuburb%20in%20Zuid-Holland%E2%80%9D%0A*%2F%0A%5Bout%3Ajson%5D%5Btimeout%3A25%5D%3B%0A%2F%2F%20fetch%20area%20%E2%80%9CZuid-Holland%E2%80%9D%20to%20search%20in%0Aarea(3600065606)-%3E.searchArea%3B%0A%2F%2F%20gather%20results%0A(%0A%20%20%2F%2F%20query%20part%20for%3A%20%E2%80%9Cplace%3Dsuburb%E2%80%9D%0A%20%20node%5B%22place%22%3D%22suburb%22%5D(area.searchArea)%3B%0A%20%20way%5B%22place%22%3D%22suburb%22%5D(area.searchArea)%3B%0A%20%20relation%5B%22place%22%3D%22suburb%22%5D(area.searchArea)%3B%0A)%3B%0A%2F%2F%20print%20results%0Aout%20body%3B%0A%3E%3B%0Aout%20skel%20qt%3B) the 'suburb density' is much higher, and what I propose here would not have any effect, as long as suburbs are rendered at lower priority than towns.

![ZL11](https://cloud.githubusercontent.com/assets/2826336/21783300/04ed926e-d6b7-11e6-9539-c1d5b339d195.JPG)
![ZL12](https://cloud.githubusercontent.com/assets/2826336/21783303/04f050a8-d6b7-11e6-8ae6-e521ad0cd7a4.JPG)
![ZL13](https://cloud.githubusercontent.com/assets/2826336/21783299/04ed8dd2-d6b7-11e6-9b34-d587c3414d9b.JPG)
![ZL11 (transport)](https://cloud.githubusercontent.com/assets/2826336/21783302/04ee1702-d6b7-11e6-9b5f-646ca64238fa.JPG)
![ZL12 (transport)](https://cloud.githubusercontent.com/assets/2826336/21783301/04ed7db0-d6b7-11e6-93d2-9262b7dcc9cc.JPG)

**Comment by jojo4u** [comment]
Place=suburb would need a better specification for this change. Currently it is used for a very wide range of subdivisions. I drafted a hierarchy of urban place subdivisions. Suburb would only be used in cities. Towns would use the smaller place=quarter. https://wiki.openstreetmap.org/wiki/User:Jojo4u/place_hierarchy This draft was [presented in german forum](https://forum.openstreetmap.org/viewtopic.php?id=55895) but did not gain any response :/

**Comment by ligfietser** [comment]
Please render place=quarter. It is a missing link between neighbourhood's and suburbs, in the Netherlands it would be a good tag to map the difference between buurten (neigbourhoods) and wijken (districts) but until now it isnt't rendered at all.

**Comment by pnorman** [comment]
This issue is about zoom thresholds for suburb, not adding quarter which is covered by #798

**Comment by ArcticGnome** [comment]
I agree. I think all three city subdivisions (neighbourhoods/quarters/suburbs) need to be larger. If that would cause problems, we need a level above suburb.

For example, I'm trying to label parts of the city of Edmonton, Canada. But the rendering size of the names of neighbourhoods would be smaller than features in the neighbourhood, like parks. So, I've called neighbourhoods "quarters" (like [Ekota](https://www.openstreetmap.org/relation/7904765)) and I've called quarters "suburbs" (like [Knottwood](https://www.openstreetmap.org/relation/9968537)). However, that leaves me with no label for the next level up (like [Mill Woods](https://www.openstreetmap.org/relation/9573853)).

In some cities, editors get around this problem by labelling the larger city subdivisions as "towns", which render at map level 11. But that solution is technically wrong for a place like Mill Woods, which is definitely a suburb of Edmonton and not its own town.

**Comment by jeisenbe** [comment]
@ArcticGnome please do not tag places improperly because of our rendering.

>  the rendering size of the names of neighbourhoods would be smaller than features in the neighbourhood, like parks

Do you mean that the font size is a smaller point than the font used for a large park? Can you show an example of where this is a problem with the correct tagging? 

I see "[Mill Woods Sports Park](https://www.openstreetmap.org/way/486655587#map=15/53.4542/-113.4398)" near this area - it is about 1/2 the size of the Ekota neighborhood - is that what you meant?

**Comment by mboeringa** [comment]
> So, I've called neighbourhoods "quarters" (like [Ekota](https://www.openstreetmap.org/relation/7904765)) and I've called quarters "suburbs" (like [Knottwood](https://www.openstreetmap.org/relation/9968537)). However, that leaves me with no label for the next level up (like [Mill Woods](https://www.openstreetmap.org/relation/9573853)).

This is typical "tagging-for-the-renderer". I understand the desire, but I would recommend you to change it back to the original tagging. Not doing so, will make sure any other map than this one displays the data wrong because you entered it wrong.

**Comment by ArcticGnome** [comment]
> I see "[Mill Woods Sports Park](https://www.openstreetmap.org/way/486655587#map=15/53.4542/-113.4398)" near this area - it is about 1/2 the size of the Ekota neighborhood - is that what you meant?

> This is typical "tagging-for-the-renderer". I understand the desire, but I would recommend you to change it back to the original tagging. Not doing so, will make sure any other map than this one displays the data wrong because you entered it wrong.

I was the person who originally gave labels to Edmonton neighbourhoods. One could make reasonable arguments for calling them either neighbourhoods or quarters. I chose to call them place=quarter because the text for neighbourhoods is so small. In your example, the text for Mill Woods Sports Park and Ekota are reasonably-sized relative to each other now, while Ekota is tagged as a quarter. But if I had used place=neighbourhood instead, the text for Ekota would have been too small.

Calling Edmonton's smallest divisions place=quarter isn't wrong, per se. The problem is that it only leaves me with one rank above. I called Knottwood (the area that Ekota is in) place=suburb. But I have nothing to call Mill Woods (the area that Knottwood is in). Now, I have Mill Woods tagged as place=borough, but as I understand it, there is a push to replace all place=borough tags with place=suburb tags.

I've seen a similar problem play out in other cities. For example, in London, England, OSM calls the city's top-level subdivisions place=town, even though they are part of the city.  Subdivisions of these "towns" are called place=suburb, and London makes little or no used of the place=neighbourhood tag.
